BuildingBlocks/src/BuildingBlocks.SDK/Endpoints/Common/EndpointClient.cs

```diff
@@ -108,7 +108,19 @@ private async Task<ApiResponse<T>> Execute<T>(HttpRequestMessage request)
             responseContent = new MemoryStream(Encoding.UTF8.GetBytes(EMPTY_RESULT));
         }
 
-        var deserializedResponseContent = JsonSerializer.Deserialize<ApiResponse<T>>(responseContent, _jsonSerializerOptions);
+        ApiResponse<T>? deserializedResponseContent;
+        try
+        {
+            deserializedResponseContent = JsonSerializer.Deserialize<ApiResponse<T>>(responseContent, _jsonSerializerOptions);
+        }
+        catch (JsonException)
+        {
+            deserializedResponseContent = new ApiResponse<T>
+            {
+                Result = default
+            };
+        }
```

Comment:
Why do you think this is a good solution?
It just covers the issue, and the user will get a seemingly valid response even though there was an error.